### ISSUE TITLE
Chapter Progress/Statistics (size, time-to-read, etc)?

### ISSUE THREAD

**Issue by vinoff** [baskerville/plato#202]
On Kobo's software, there is a very handy feature which is the chapter progress/statistics, in the form of a bar-chart. It is only available for kepub if I remember correctly.

I would love to have something like that on Plato. As it is, I feel a bit "lost" reading on Plato as I like knowing if I am nearing the end of a chapter or knowing the size of the next few chapters and whatnot.

**Comment by thataboy** [comment]
I second this. In lieu of fancy bar charts, a simple progress bar on bottom showing book progress along with page turns left in chapter would be most helpful

**Comment by tootbrute** [comment]
Yes, but let's not turn Plato into the bloated beast that KOReader is. The clean uncluttered aesthetic is what sets Plato apart.

**Comment by Tim-H-59** [comment]
For me this is only feature holding me back to switching to Plato, but I agree it should not be a default in order to keep Plato uncluttered.

**Comment by it0na72** [comment]
would love to see this implemented. something as simple as the number of pages left in a chapter would be really nice.
